pandas/tests/arrays/categorical/test_operators.py

```diff
@@ -186,10 +185,30 @@ def test_comparison_with_unknown_scalars(self):
         tm.assert_numpy_array_equal(cat != 4,
                                     np.array([True, True, True]))
 
+    def test_comparison_of_ordered_categorical_with_nan_to_scalar(
+        self, compare_operators_no_eq_ne):
+        # https://github.com/pandas-dev/pandas/issues/26504
+        # BUG: fix ordered categorical comparison with missing values (#26504 )
+        # and following comparisons with scalars in categories with missing
+        # values should be evaluated as False
+
+        cat = Categorical([1, 2, 3, None], categories=[1, 2, 3], ordered=True)
+
+        assert getattr(cat, compare_operators_no_eq_ne)(2)[-1] == False
```

Comment:
This will fail, since now we test compare_operators_no_eq_ne which includes <, <=, >, >=. I need to list all four expected results.

Reply:
you can do this

```
with warnings.catch_warnings():
            warnings.simplefilter("ignore", RuntimeWarning)
            expected = getattr(np.array(cat), op)(2)
```

which will result in a numpy array (note that we suppress the warnings because numpy does the correct comparison with NaN but it shows a warning)

Reply:
Thanks for suggestion, Jeff! I struggled to make pytest.mark.parametrize work.